phase2/ansible/playbooks/roles/master/templates/kube-apiserver.jsonnet

```diff
@@ -15,32 +18,31 @@ function(cfg)
       containers: [
         {
           name: "kube-apiserver",
-          image: "%(docker_registry)s/kube-apiserver:%(kubernetes_version)s" % cfg.phase2,
+          image: "%(docker_registry)s/hyperkube-amd64:%(kubernetes_version)s" % cfg.phase2,
           resources: {
             requests: {
               cpu: "250m",
             },
           },
-          command: [
-            "/bin/sh",
-            "-c",
-            |||
-              /usr/local/bin/kube-apiserver \
-                --address=127.0.0.1 \
-                --etcd-servers=http://127.0.0.1:2379 \
-                --cloud-provider=%(cloud_provider)s \
-                --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,ResourceQuota \
-                --service-cluster-ip-range=10.0.0.0/16 \
-                --client-ca-file=/srv/kubernetes/ca.pem \
-                --tls-cert-file=/srv/kubernetes/apiserver.pem \
-                --tls-private-key-file=/srv/kubernetes/apiserver-key.pem \
-                --secure-port=443 \
-                --allow-privileged \
-                --v=4
-            ||| % cfg.phase1,
-            # --basic-auth-file=/srv/kubernetes/basic_auth.csv \
-            # --token-auth-file=/srv/kubernetes/known_tokens.csv \
-          ],
+          command: build_params([
+            [
+              "/hyperkube",
+              "apiserver",
+              "--address=127.0.0.1",
+              "--etcd-servers=http://127.0.0.1:2379",
+              "--cloud-provider=%s" % cfg.phase1.cloud_provider,
+              "--admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,ResourceQuota",
+              "--service-cluster-ip-range=10.0.0.0/16",
```

Comment:
The 10.246.0.0/24 is the master node's container bridge ip addresses. The node's IP is allocated on the 10.240.0.0/16.